src/lib/components/input/index.tsx

```diff
@@ -29,7 +32,7 @@ export default React.forwardRef(
     }: InputProps,
     ref?: React.ForwardedRef<HTMLInputElement>
   ) => {
-    const uniqueId = id ?? uuidv4();
+    const [uniqueId] = useState(id ?? generateUniqueId());
```

Comment:
Why do we need a state here? Is only having a computed value not enough?

Reply:
Is there any harm in dropping that id generation altogether? As far as I can tell the id is forwarded to the underlying `input` if an id would be needed and an `htmlFor` prop could be `similarly` exposed instead.

Reply:
If it is an a11y concern then I would argue that a manually set `id` is a lot more helpful to screen readers etc, than a randomly generated string.

Reply:
@kimkwanka we could go with the implicit label route, but remember reading that using ids is better than implicit, but I might be mistaken.

Reply:
https://dev.to/thejaredwilcurt/the-truth-behind-implicitexplicit-form-labels-2e10

Seems like they should in theory be equal, but apparently there are some screen readers that do not work nicely with implicit labels. Guess we can go with random labels then :)

Reply:
Still not sure I like the potential overlap with Math.random here but then again I am not sure if alternatives like [nanoid](https://github.com/ai/nanoid) have the same stability issues as uuid

Reply:
Agreed @kimkwanka 
Maybe we can do it in two steps.
Merge this to fix the uuid dependency issue and then we could try to go the implicit label route or discuss it.